src/commands/dev/lib/dev-builder.ts

```diff
@@ -82,11 +83,15 @@ async function createBuildProcess(
     nodeBinPromise,
     builderModulePathPromise
   ]);
+  let PATH = `${dirname(execPath)}:${process.env.PATH}`;
+  if (yarnPath) {
+    PATH = `${yarnPath}:${PATH}`;
```

Comment:
Ditto (I realize that it was already using `:` before, but might as well correct it while we're here).

Reply:
Fixed